Saurabh taking over Shreya P taking over for Vamsidhar taking over for Pujitha taking over for Nishita_registration_confirmation_popup_modal

nishitag5 opened the pull request:
<img width="473" height="732" alt="image" src="https://github.com/user-attachments/assets/2744ac6b-f9f9-4cf7-88d9-c19b74a5120c" />
<img width="466" height="317" alt="image" src="https://github.com/user-attachments/assets/90045f39-cf21-4ae0-9daf-562eaf9b5d7c" />

# Description
Implemented the event registration confirmation popup modal after successful registration.

## Related PRS (if any):
No related PRs
…

## Main changes explained:
- Updated the routes.js file to add the URL mapping for the event registration confirmation modal.
- Added the component for the confirmation popup.
…

## How to test:
1. check into current branch
2. do npm install and ... to run this PR locally
3. Clear site data/cache
4. go to the URL http://localhost:3000/communityportal/activities/registration
5. Verify that the UI for the page.

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/37e014f5-3f0e-4cf0-a007-20985f1ab019


## Note:
Note that the dark view is only implemented for the popup.

nishitag5 committed:
adding Registration confirmation pop up

palankigreeshma1109 reviewed (dismissed):
I have reviewed this PR and followed every step and navigated to http://localhost:3000/communityportal/activities/registration.
Here are the screenshots available for your reference.
1.In Light mode
![Screenshot (156)](https://github.com/user-attachments/assets/bc293745-ed2a-4c96-90b6-f1b35fa52d93)

2.Dark mode
![Screenshot (157)](https://github.com/user-attachments/assets/f23394fa-74e0-4c7e-881b-09fd051acbd3)

cgomezhub reviewed (approved):
All right!. The popup functionality  and dark mode  are   working correctly, it opens and closes to click events. others functionality inside the modal have not been implemented.
![Screenshot 2025-04-12 081009](https://github.com/user-attachments/assets/a53ecd43-60b5-411e-aab0-b394c9bca7a3)
![Screenshot 2025-04-12 081146](https://github.com/user-attachments/assets/033882f9-6f71-4c11-84ab-6d966a0f2278)

pavanputti reviewed (commented):
I tested the fix by checking out the current branch, running npm install, and starting the app locally after clearing the site data and cache. I navigated to http://localhost:3000/communityportal/activities/registration and verified that the UI for the page displayed correctly.

<img width="1670" alt="Screenshot 2025-04-12 at 11 02 52 AM" src="https://github.com/user-attachments/assets/9ae3de74-7222-4949-ab95-107c879ab070" />

<img width="1670" alt="Screenshot 2025-04-12 at 11 00 52 AM" src="https://github.com/user-attachments/assets/dc8a2ab2-c714-41de-a3fa-6e443a1aaa95" />

rsripathi2906 reviewed (dismissed):
Verified the popup functionality and dark mode both are working correctly.
1. Light theme
![image](https://github.com/user-attachments/assets/9d642b8d-34aa-4918-bc77-1d3e9576eaed)
2.Dark theme 
![Screenshot 2025-04-12 100046](https://github.com/user-attachments/assets/a541ec43-c391-4448-a12b-49675514a887)

JuliaHa0902 commented on src/components/CommunityPortal/RegistrationConfirmation/RegistrationPopup.jsx:
```diff
@@ -0,0 +1,47 @@
+        </p>
+        <div className="popup-content">
+          <strong>Event Name</strong>
+          <p className="event-details">(Click for more details)</p>
```
Can you change this to a lighter blue in dark mode please? Maybe choose the same color as the h2. The dark blue color of the link can hardly be seen in the dark background.
<img width="344" alt="dark text on dark background" src="https://github.com/user-attachments/assets/909a4a57-8816-48c2-ba6f-d26a7dda8977" />

JuliaHa0902 reviewed (commented):
Please see the request change below.

JuliaHa0902 reviewed (changes requested):
I think the text "Click for more details" should have a lighter color in dark mode. Currently now it has a dark blue color, which is hard to be seen on a dark background.
<img width="344" alt="dark text on dark background" src="https://github.com/user-attachments/assets/0724b44f-9140-48df-af2e-f41ef950ac53" />

Otherwise, the modal works great. When I clicked on the Registration button, the modal showed
![Screenshot 2025-04-12 114312](https://github.com/user-attachments/assets/0bb30a87-904c-4cb3-a1be-99c870bc4f38)
When I switched to dark mode, it changed to dark mode
![Screenshot 2025-04-12 122328](https://github.com/user-attachments/assets/e198317e-ce64-476e-95f7-c43051d65524)

KH07 commented:
Tested and functioned as intended. I checked out the current branch, ran npm install, and started the app locally. On the registration page, a modal pops up with confirmation after clicking the ‘Register’ button.

<img width="1558" alt="PR#3405" src="https://github.com/user-attachments/assets/7e2d5062-1145-4f83-88e8-0ae569525fc3" />

vaibhavkoladiya reviewed (dismissed):
Checked out the current branch, ran npm install and started the app. Cleared site data/cache and verified that the registration activities page UI loads correctly and works properly in dark mode. Approving the PR.

![Screenshot 2025-04-12 142544](https://github.com/user-attachments/assets/44309090-40c4-4825-a7c1-bf59b2b57751)

![Screenshot 2025-04-12 142608](https://github.com/user-attachments/assets/e753e931-a4c4-4f10-aa22-712b38e13fc5)

sravankumarbodakonda reviewed (dismissed):
I have tested the pull request using the provided steps, and everything is working as expected in both light and dark modes.
![Screenshot 2025-04-13 at 9 59 25 PM](https://github.com/user-attachments/assets/8f293810-5203-4ae0-ad69-079e39b82b3d)
![Screenshot 2025-04-13 at 9 59 48 PM](https://github.com/user-attachments/assets/eedb38f9-aab4-4c7b-b75d-efee8df36733)

rahulprasad2823 reviewed (approved):
I have tested the PR, everything looks good.

![Screenshot (6)](https://github.com/user-attachments/assets/a5cff803-fd22-4b24-9f17-b4e3fe1e9f39)
![Screenshot (7)](https://github.com/user-attachments/assets/77e89e83-8ffb-4e89-8ca7-d9fcba899f11)

gopikalakshmia commented:
Verified the PR:event registration confirmation popup, it is working as demonstrated in the demo.
Dark Mode
<img width="468" alt="image" src="https://github.com/user-attachments/assets/7a6d91c0-35fc-4e96-bd84-71ad3135d4ab" />
Light Mode
<img width="468" alt="image" src="https://github.com/user-attachments/assets/60b59de7-0460-4848-ad8e-5c055a33ebce" />

gopikalakshmia reviewed (dismissed):
Verified the event registration confirmation popup; it is working as demonstrated in the demo.
Dark Mode
<img width="468" alt="image" src="https://github.com/user-attachments/assets/a119763d-2316-401f-a194-28f8006e1cb6" />
Light Mode
<img width="468" alt="image" src="https://github.com/user-attachments/assets/19f12227-a149-49d8-a8fe-9733d324c18a" />

Jitesh2106 reviewed (dismissed):
![PR-3405](https://github.com/user-attachments/assets/5c34c3f1-70cb-4534-a74c-d348bdb55be6)
I confirmed that the event registration confirmation popup is functioning correctly and behaves as shown in the demo.

1709abhishek commented:
<img width="866" alt="Screenshot 2025-06-07 at 12 28 23 PM" src="https://github.com/user-attachments/assets/c1d35fbb-3b15-4fb7-bed8-74434b3b51cb" />
works fine

sircarmart reviewed (dismissed):
Register button works and displays modal. Formatting on modal looks great and fits well on smaller screen sizes.
<img width="2560" height="1440" alt="image" src="https://github.com/user-attachments/assets/135d72c0-18a0-4218-8b81-8279f285b94b" />
<img width="520" height="840" alt="image" src="https://github.com/user-attachments/assets/bd967663-43ad-4cc8-95ad-d335934030ad" />

Also supports dark mode, looks great!
<img width="2560" height="1440" alt="image" src="https://github.com/user-attachments/assets/98055ccd-9e9d-4852-8dc6-5d5edf585fbe" />

Rishitha1810 commented:
Pulled the branch locally, installed dependencies, and launched the app. Cleared browser cache and confirmed that the Registration Activities page displays as expected. Verified that all UI elements function correctly in dark mode. The PR is ready for approval.
<img width="1470" height="798" alt="Screenshot 2025-07-19 at 3 30 48 PM" src="https://github.com/user-attachments/assets/ac438fcb-a0ac-48e9-9311-8404905361f3" />
<img width="1470" height="798" alt="Screenshot 2025-07-19 at 3 30 56 PM" src="https://github.com/user-attachments/assets/57a5d789-f933-4dcd-84c7-4adea6373984" />
<img width="1470" height="798" alt="Screenshot 2025-07-19 at 3 31 29 PM" src="https://github.com/user-attachments/assets/e177de55-1597-4fa6-a4a9-dd08ebd8bd17" />

vamsidharpanithi committed:
Fix dark mode text contrast for event details link

vamsidharpanithi committed:
adding changes

# Conflicts:
#	src/routes.js

vamsidharpanithi committed:
Renames routes.js to routes.jsx and fixed test import

# Conflicts:
#	src/components/ForcePasswordUpdate/__tests__/ForcePasswordUpdate.test.js
#	src/components/Projects/WBS/__tests__/Projects.WBS.test.jsx
#	src/routes.jsx

vamsidharpanithi committed:
adding changes

vamsidharpanithi committed:
Fix import issues in routes.jsx - resolve TaskEditSuggestions and EmailSender imports

vamsidharpanithi committed:
Resove merge conflicts

SreePujitha01 committed:
Fixed User full name and icons to registration modal

SreePujitha01 committed:
Fixed Registration confirmation page

vamsidharpanithi committed:
Merge development into nishita_registration_confirmation_modal - resolved conflicts in routes.jsx and yarn.lock, converted CSS files to module CSS

vamsidharpanithi committed:
Fix CSS module imports after merge

vamsidharpanithi committed:
Fix remaining CSS module imports

vamsidharpanithi committed:
Fix configureStore imports and rename PermissionsManagement CSS file

vamsidharpanithi committed:
Convert RegistrationPopup.css to module CSS format

shashank-madan reviewed (approved):
Looks good to me

<img width="1898" height="1058" alt="Image" src="https://github.com/user-attachments/assets/e847c18d-3f9b-49e2-8abd-b64bc96a3f31" />

<img width="1902" height="1001" alt="Image" src="https://github.com/user-attachments/assets/fd720f9d-83d7-4c9f-9c4d-7a3241a18427" />

rohanrastogi311 reviewed (approved):
Hi Vamsidhar, 

Well done with this implementation. 

<img width="1728" height="1032" alt="Image" src="https://github.com/user-attachments/assets/1458fe24-9153-49ad-b74e-8fc41a75c43e" />
<img width="1723" height="1034" alt="Image" src="https://github.com/user-attachments/assets/e5e73ca7-c223-4e60-8120-c435311830a5" />

Commit:
fix: resolve merge conflicts with development branch

Commit:
fix: SonarCloud issues - unused imports, duplicate CSS, PropTypes

Commit:
fix: improve color contrast in RegistrationPopup to meet accessibility requirements

Commit:
fix: fix remaining contrast issues in RegistrationPopup

DeepighaJ reviewed (changes requested):
Checked out to current branch and pop up appears good in dark and light mode but click for more details link and other buttons are not clickable in both modes.

<img width="1600" height="852" alt="Image" src="https://github.com/user-attachments/assets/21f6c1b1-3f09-4e5e-a777-b44c1f20de08" />
<img width="1600" height="852" alt="Image" src="https://github.com/user-attachments/assets/c6f60539-bbc9-4edd-ab9e-8f4df498ec29" />

Commit:
Merge origin/development and resolve conflicts

Commit:
fix: correct selector ordering for popup-dark button:disabled

Commit:
fix: page background now responds to dark/light mode toggle

Commit:
fix: replace window.alert with globalThis.alert for SonarCloud

DeepighaJ reviewed (approved):
Checked out to current branch and tested PR locally.
The Register button displays the popup as expected in both dark and light mode. The links are separate new scopes approving this PR.

<img width="1600" height="852" alt="Image" src="https://github.com/user-attachments/assets/bd43eeac-9a00-42c6-9d7d-11a4910b4089" />
<img width="1600" height="852" alt="Image" src="https://github.com/user-attachments/assets/38353de5-3136-4b13-94b8-d06ae91679dc" />
<img width="1600" height="852" alt="Image" src="https://github.com/user-attachments/assets/ed24b782-cb7d-4467-baed-5d9676cacc8e" />
